installer/install.iss

```diff
@@ -548,6 +549,32 @@ begin
     end;
 end;
 
+function ListOpenSSHExecutables():TArrayOfString;
+var
+    Executables:TArrayOfString;
+begin
+    // Amalgamation of executables delivered by common OpenSSH implementations:
+    // https://github.com/PowerShell/Win32-OpenSSH/releases
+    // https://packages.msys2.org/package/openssh?repo=msys&variant=x86_64
+    SetArrayLength(Executables,15);
+    Executables[0]:='findssl.sh'; // msys2 only
+    Executables[1]:='scp.exe';
+    Executables[2]:='sftp-server.exe';
+    Executables[3]:='sftp.exe';
+    Executables[4]:='ssh-add.exe';
+    Executables[5]:='ssh-agent.exe';
+    Executables[6]:='ssh-copy-id'; // msys2 only
+    Executables[7]:='ssh-keygen.exe';
+    Executables[8]:='ssh-keyscan.exe';
+    Executables[9]:='ssh-keysign.exe'; // msys2 only
+    Executables[10]:='ssh-pkcs11-helper.exe'; // msys2 only
+    Executables[11]:='ssh-shellhost.exe'; // Win32 only
+    Executables[12]:='ssh-sk-helper.exe'; // msys2 only
+    Executables[13]:='ssh.exe';
+    Executables[14]:='sshd.exe';
+    Result:=Executables;
```

Comment:
The full list of files is a bit longer, and can be obtained in `release.sh` via

```sh
comm -12 <(echo "$LIST" | sort) <(pacman -Ql openssh | sed -n 's|^openssh /\(.*[^/]\)$|\1|p' | sort)
```

When I run it here, it will produce these lines:

```
etc/ssh/moduli
etc/ssh/ssh_config
etc/ssh/sshd_config
usr/bin/findssl.sh
usr/bin/scp.exe
usr/bin/sftp.exe
usr/bin/ssh-add.exe
usr/bin/ssh-agent.exe
usr/bin/ssh-copy-id
usr/bin/ssh-keygen.exe
usr/bin/ssh-keyscan.exe
usr/bin/ssh.exe
usr/bin/sshd.exe
usr/lib/ssh/sftp-server.exe
usr/lib/ssh/ssh-keysign.exe
usr/lib/ssh/ssh-pkcs11-helper.exe
usr/lib/ssh/ssh-sk-helper.exe
usr/share/licenses/openssh/LICENCE
```

If I were you, I would use this invocation as part of writing a function to `config.iss` similar to how `SetSystemConfigDefaults()` is generated. Something along these lines:

```sh
openssh_deletes="$(comm -12 <(echo "$LIST" | sort) \
        <(pacman -Ql openssh | sed -n 's|^openssh /\(.*[^/]\)$|\1|p' | sort) |
    sed -e 'y/\//\\/' -e "s|.*|    if not DeleteFile(AppDir+'\\\\&') then\n        Result:=False;|")"
inno_defines="$inno_defines$LF[Code]${LF}function DeleteOpenSSHFiles():Boolean;${LF}begin$LF    Result:=True;"
inno_defines="$inno_defines$LF$openssh_deletes${LF}end;$LF#define DELETE_OPENSSH_FILES DeleteOpenSSHFiles$LF"
```

And then in `install.iss` at the appropriate point do something like

```innosetup
#ifdef DELETE_OPENSSH_FILES
    if RdbSSH[GS_ExternalOpenSSH].Checked then begin
        WizardForm.StatusLabel.Caption:='Removing bundled Git OpenSSH binaries';
        if not DELETE_OPENSSH_FILES() then
            LogError('Failed to remove OpenSSH file(s)');
end;
```

Note: this is fully untested, therefore I expect that you will have to work on this for a bit until it does what it is expected to do.

Reply:
Will have a look, thanks!